src/scripting/lua_map.cc

```diff
@@ -4259,7 +4304,9 @@ int LuaTerrainDescription::get_fertility(lua_State* L) {
 /* RST
    .. attribute:: humidity
 
-         (RO) the :class:`uint` humidity value for this terrain
+      (RO) The humidity value for this terrain as.
```

Comment:
The sentence seems. 

                    ^ incomplete ;)

Reply:
fixed.